src/EmojiTranslatorContainer.js

```diff
@@ -5,14 +5,68 @@ import Stories from './components/Stories';
 export default class EmojiTranslatorContainer extends Component {
   constructor() {
     super()
+    this.state = {
+      originalContent: '',
+      creator: '',
+      translatedContent: '',
+      stories: []
+    }
+  }
+
+  handleTranslate(event) {
+    this.setState({
+      originalContent: event.target.value,
+      translatedContent: event.target.value.toUpperCase()
+    })
+    console.log('from handleTranslate: ', this.state.originalContent);
+  }
 
+  handleCreator(event) {
+    this.setState({
+       creator: event.target.value
+    })
   }
 
-  render(){
+  handleSubmit(event) {
+    event.preventDefault()
+    fetch('http://localhost:3000/stories', {
```

Comment:
i might move this into a separate API function to make it more readable. `createStory(storyData).then( story => console.log(story))`